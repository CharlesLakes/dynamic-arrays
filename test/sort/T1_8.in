10
69 51 75 74 44 34 29 18 92 50