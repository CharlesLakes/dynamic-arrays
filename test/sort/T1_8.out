18 29 34 44 50 51 69 74 75 92